app/pyproject.toml

```diff
@@ -70,7 +70,6 @@ db-migrate-down = "src.db.migrations.run:down"
 db-migrate-down-all = "src.db.migrations.run:downall"
 ingest-guru-cards = "src.ingest_guru_cards:main"
 ingest-policy-pdfs = "src.ingest_policy_pdfs:main"
```

Comment:
Since you removed the file, remove this line
```suggestion
```

Reply:
@yoomlam , I was seeing a few new test failures after removing `ingest_policy_pdfs.py` namely in `test_pdf_postprocess.py`. Tried to do some finer-grained removal of assertions from `test_ingest_policy_pdfs.py` in this most recent commit and tests are passing now.